selectel/tickets/app.py

```diff
@@ -0,0 +1,140 @@
+import os
+
+from datetime import datetime
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from flask import request
+from config import TICKET_STATUS_OPEN, TICKET_STATUS_CLOSED, TICKET_STATUS_ANSWERED, TICKET_STATUS_WAITING_FOR_ANSWER, \
```

Comment:
Длинные импорты лучше писать в столбец
```suggestion
from config import (
    TICKET_STATUS_OPEN,
    TICKET_STATUS_CLOSED,
    TICKET_STATUS_ANSWERED,
    TICKET_STATUS_WAITING_FOR_ANSWER,
    TICKET_STATUSES,
    REDIS_TICKET_COMMENTS_KEY_TEMPLATE,
)
```